config/initializers/pdp.rb

```diff
@@ -0,0 +1,2 @@
+# Initialize Atmosphere pdp class to DefaultPdp
+Atmosphere::at_pdp_class = Atmosphere::DefaultPdp
```

Comment:
Are you sure we should put it here? If engine is included in dependent project there can be initializers order loading problem.